src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function ADD(num) {
```

Comment:
```suggestion
  return function adder(num) {
```

Usually, we use uppercase for naming constants. Try not to name functions in uppercase